src/diffusers/commands/env.py

```diff
@@ -12,29 +12,64 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import platform
 from argparse import ArgumentParser
+from importlib import util
 
 import huggingface_hub
 
 from .. import __version__ as version
-from ..utils import is_accelerate_available, is_torch_available, is_transformers_available, is_xformers_available
+from ..utils import (
+    is_accelerate_available,
+    is_flax_available,
+    is_google_colab,
+    is_notebook,
+    is_peft_available,
+    is_safetensors_available,
+    is_torch_available,
+    is_transformers_available,
+    is_xformers_available,
+)
 from . import BaseDiffusersCLICommand
 
 
 def info_command_factory(_):
     return EnvironmentCommand()
 
 
+def download_command_factory(args):
+    return EnvironmentCommand(args.accelerate_config_file)
+
+
 class EnvironmentCommand(BaseDiffusersCLICommand):
     @staticmethod
     def register_subcommand(parser: ArgumentParser):
         download_parser = parser.add_parser("env")
         download_parser.set_defaults(func=info_command_factory)
+        download_parser.add_argument(
+            "--accelerate-config_file",
+            default=None,
+            help="The accelerate config file to use for the default values in the launching script.",
+        )
+        download_parser.set_defaults(func=download_command_factory)
+
+    def __init__(self, accelerate_config_file, *args) -> None:
+        self._accelerate_config_file = accelerate_config_file
 
     def run(self):
         hub_version = huggingface_hub.__version__
 
+        safetensors_version = "not installed"
+        if is_safetensors_available():
+            import safetensors
+
+            safetensors_version = safetensors.__version__
+        elif util.find_spec("safetensors") is not None:
+            import safetensors
```

Comment:
Why is this needed?

Reply:
I forgot to mention. I took this directly from [`transformers`](https://github.com/huggingface/transformers/blob/243d0de9971d953b2a69636fba0420fb56cd42e4/src/transformers/commands/env.py#L62-L65).

Reply:
That's fine. It's okay to make things leaner here 😉